keyboards/kiiboom/cybrix16/rules.mk

```diff
@@ -0,0 +1,7 @@
+
+MCU_LDSCRIPT = STM32F103xB
+ENCODER_MAP_ENABLE = yes

```

Comment:
Encoder map should only be enabled at the keymap level, never at the keyboard level.